src/runtime/pay-flow.js

```diff
@@ -38,15 +41,28 @@ import {
 } from '../utils/json';
 import {AnalyticsEvent} from '../proto/api_messages';
 
+
+/**
+ * String values input by the publisher are mapped to the number values.
+ * @enum {number}
```

Comment:
This is pretty critical, but this cannot be an enum. Enum's keys (but not values) are obfuscated. Thus, if you try to do `ReplaceSkuProrationModeMapping['THE_ACTUAL_VALUE']` - it won't have the right mapping. Thus, this has to be an object, and the keys must be in quotes.

Reply:
done